controllers/homework.js

```diff
@@ -398,7 +398,7 @@ router.delete('/:id/file', (req, res, next) => {
 router.get('/submit/:id/import', getImportHandler('submissions'));
 router.patch('/submit/:id', getUpdateHandler('submissions'));
 router.post('/submit', getCreateHandler('submissions'));
-router.delete('/submit/:id', getDeleteHandler('submissions', true));
+router.delete('/submit/:id', getDeleteHandler('submissions'));
```

Comment:
I'm irritated that you removed the true here for two reasons:
1) The get route below is used by the delete link. As far as I see it, this delete route is unused can be removed entirely.
2) The redirect does not work without the true. So you must leave it in the get route.